lib/active_merchant/billing/gateways/braintree/braintree_common.rb

```diff
@@ -18,4 +23,9 @@ def scrub(transcript)
       gsub(%r((&?ccnumber=)\d*(&?)), '\1[FILTERED]\2').
       gsub(%r((&?cvv=)\d*(&?)), '\1[FILTERED]\2')
   end
+
+  def supports_network_tokenization?
+    response = authorize(1, ActiveMerchant::Billing::NetworkTokenizationCreditCard.test_credit_card)
+    response.params['braintree_transaction']['processor_response_code'] != ERROR_CODES.fetch(:payment_instrument_not_supported)
```

Comment:
What happens in case of a connection error or 5xy response and this body isn't there? It is appropriate that we let that bubble up (probably, yes)? I'm assuming an error will be raised before we get to the hash traversal, yeah?